projects/packages/videopress/src/class-block-editor-content.php

```diff
@@ -21,31 +21,67 @@ public static function init() {
 			return;
 		}
 
-		add_filter( 'default_content', array( static::class, 'video_block_by_guid' ), 10, 2 );
+		add_shortcode( 'jetpack_videopress', array( static::class, 'videopress_block_shortcode' ) );
+		add_filter( 'default_content', array( static::class, 'videopress_video_block_by_guid' ), 10, 2 );
 	}
 
 	/**
-	 * Generates a video block content with the given guid
+	 * VideoPress embed shortcode
+	 *
+	 * Example use:
+	 * [jetpack_videopress guid=tLvEwHYZ width=560 height=315]
+	 *
+	 * @param array $atts Shortcode attributes.
+	 *
+	 * @return string html
+	 */
+	public static function videopress_block_shortcode( $atts ) {
+		$atts = shortcode_atts(
+			array(
+				'guid'   => '', // string.
+				'width'  => 560, // int.
+				'height' => 315, // int.
+			),
+			$atts,
+			'jetpack_videopress'
+		);
+
+		$guid = sanitize_text_field( wp_unslash( $atts['guid'] ) );
+
+		if ( empty( $guid ) ) {
+			return '<!-- error: missing VideoPress video ID -->';
+		}
+
+		$width  = (int) $atts['width'];
+		$height = (int) $atts['height'];
+		$src    = esc_url( 'https://videopress.com/embed/' . $guid );
+
+		wp_enqueue_script( 'videopress-iframe', 'https://videopress.com/videopress-iframe.js', array(), Package_Version::PACKAGE_VERSION, true );
+
+		return '<iframe src="' . $src . '" frameborder="0" allowfullscreen allow="clipboard-write" width="' . $width . '" height="' . $height . '"></iframe>';
```

Comment:
```suggestion
		$block_template =
'<figure class="wp-block-videopress-video wp-block-jetpack-videopress jetpack-videopress-player">' .
	'<div class="jetpack-videopress-player__wrapper">' .
		'<iframe ' .
			'title="' . __( 'VideoPress Video Player' ) . '" ' .
			'aria-label="' . __( 'VideoPress Video Player' ) . '" ' .
			'src="%s" '.
			'width="%s"' .
			'height="%s" '.
			'frameborder="0" ' .
			'allowfullscreen data-resize-to-parent="true" allow="clipboard-write">' .
		'</iframe>' .
	'</div>' .
'</figure>';

		return sprintf( $block_template, $src, $width, $height );
```

The code above adds the CSS class that tweaks the plater in the front end, trying to follow the block handling approach.